addons/hr_org_chart/views/hr_views.xml

```diff
@@ -6,9 +6,17 @@
         <field name="inherit_id" ref="hr.view_employee_form"/>
         <field name="arch" type="xml">
             <div id="o_work_employee_main" position="after">
-                <div id="o_employee_right">
-                    <h4 class="o_org_chart_title mb16 mt0">Organization Chart</h4>
-                    <field name="child_ids" widget="hr_org_chart" readonly="1"/>
+                <div id="o_employee_right" class="col-lg-4 px-0 ps-lg-5 pe-lg-0">
+                    <table class="o_group o_inner_group">
+                        <tbody>
+                            <tr>
+                                <td class="pe-lg-0">
```

Comment:
I'm confused: why adding `<group>` like HTML? And specially for a 1line/1cell table? :thinking:

Reply:
To ensure margin consistency with other formView elements.
I don't like it neither but it's the cleanest approach until we refactor the formView design :/
![image](https://user-images.githubusercontent.com/8558351/182651790-6b646d44-ecca-446e-8a70-22783d778499.png)